### DISCUSSION TITLE
Lightning Flash collaboration

### DISCUSSION THREAD

**Discussion by tchaton** [pykale/pykale#290] [Ideas]
# 🚀 Feature Request

Dear team behind pykale,

Awesome library, really neat initiative.
I am leading the PyTorch Lightning Team.
I believe there is lot of overlapping with Lightning Flash: https://github.com/PyTorchLightning/lightning-flash.

We wondered if you would be interested in some form of collaboration ?

Best,
T.C

## Motivation

**Is your feature request related to a problem? Please describe.**

## Pitch

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Are you willing to open a pull request?** (We LOVE contributions!!!)

**Comment by haipinglu** [comment]
Thank you for reaching out @tchaton! PyKale was largely inspired by PyTorch Lightning so of course, we would be happy explore possible collaborations. How would you like to proceed?
